src/backend/base/langflow/api/v1/chat.py

```diff
@@ -510,7 +512,7 @@ async def build_vertex(
             lock = chat_service._async_cache_locks[flow_id_str]
             vertex_build_result = await graph.build_vertex(
                 vertex_id=vertex_id,
-                user_id=current_user.id,
+                user_id=str(current_user.id),
```

Comment:
There was a typing error as `current.id` is an `UUID` and `build_vertex` takes a string.
I think it's OK to transform the UUID to string as `self._uuid` can be both string and `UUID` in `BaseComponent`.
WDYT @ogabrielluiz ?

Reply:
I agree